src/components/VesyolkinAS/React-Sweeper-master/src/App.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-alert */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-plusplus */
+/* eslint-disable react/button-has-type */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, { useState } from 'react';
+import styles from './styles.css';
```

Comment:
Чекнул, в файл CSS после билда всё ещё добавляются классы с неуникальными названиями, но если сделать так, то всё работает.
```suggestion
import styles from './Minesweeper.module.css';
```
Файл стилей соответственно переименовать.